app/helpers/editorial_helper.rb

```diff
@@ -1,7 +1,9 @@
 module EditorialHelper
 
   def form_type(node_clazz)
-    "#{node_clazz.name}Form".constantize
+    # Because of development mode autoloading, the class may not exist until
+    # we call constantize. Hence just catching the exception.
+    "#{node_clazz.name}Form".constantize rescue NodeForm
```

Comment:
Would it be better to be explicit? We could make a `CustomTemplateNodeForm` and have that extend `NodeForm`.